test/framework/src/main/java/org/opensearch/test/OpenSearchTestCase.java

```diff
@@ -290,6 +290,7 @@ private static void setTestSysProps() {
 
         // Enable Netty leak detection and monitor logger for logged leak errors
         System.setProperty("io.netty.leakDetection.level", "paranoid");
+        System.setProperty("opensearch.experimental.feature.telemetry.enabled", "true");
```

Comment:
Please remove, we have a feature flag mecanism to turn features on / off

Reply:
This is again to keep it enable by default to enforce the strict check for all the spans.

Reply:
The `OpenSearchIntegTestCase` has the features flags, please use that, not a system property (this is where the plugin is installed)